conf/rest/9.12.0/disk.yaml

```diff
@@ -30,6 +29,7 @@ endpoints:
       - ^^uid => uid
       - sectors_read => stats_sectors_read
       - sectors_written => stats_sectors_written
+      - disk_io_kbps_total => stats_io_kbps
```

Comment:
stats.throughput is in bytes in public rest. We'll need plugin to covert to kbps. We can do that later.